content/pt-br/docs/concepts/overview/working-with-objects/object-management.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Gerenciamento de Objetos do Kubernetes
+content_type: concept
+weight: 15
+---
+
+<!-- overview -->
+A ferramenta de linha de comando `kubectl` suporta várias maneiras diferentes de criar e 
+gerenciar objetos do Kubernetes. Este documento fornece uma visão geral das diferntes
+abordagens. Leia o [livro do Kubectl](https://kubectl.docs.kubernetes.io) para detalhes
+do gerenciamento de objetos pelo *Kubectl*.
+
+<!-- body -->
+
+## Técnicas de Gerenciamento
+
+{{< warning >}}
+Um objeto Kubernetes deve ser gerenciado utilizando uma única técnica. Mistura e 
+combinação de técnicas para um mesmo objeto resulta em um comportamento indefinido.
+{{< /warning >}}
+
+| Técnica de gerenciamento             | Opera em         | Ambiente recomendado | Número de agentes de escrita suportados  | Curva de aprendizado |
+|----------------------------------|----------------------|------------------------|--------------------|----------------|
+| Comandos imperativos              | Objetos ativos       | Projetos em desenvolvimento | 1+                 | Baixo         |
+| Configuração do objeto imperativo  | Arquivos individuais     | Projetos em produção    | 1                  | Moderado       |
+| Configuração da declaração do objeto | Diretórios de arquivos | Projetos em produção    | 1+                 | Alto           |
+
+## Comandos imperativos
+
+Ao usar comandos imperativos, um usuário opera diretamente em 
+objetos ativos em um *cluster*. O usuário fornece operações para
+o comando `kubectl`  como argumentos ou *flags*.
+
+Isso é a maneira recomendada para iniciar ou executar uma única tarefa
+em um *cluster*. Pois esta técnica opera diretamente em objetos ativos,
```

Comment:
Suggestion: I think "Como esta técnica" sounds better in this sentence, because it is going to explain something about the technique in the next sentence.